packages/react/src/context/slash-id-context.tsx

```diff
@@ -158,7 +176,16 @@ export const SlashIDProvider = ({
   const storageRef = useRef<Storage | undefined>(undefined);
   const sidRef = useRef<SlashID | undefined>(undefined);
   const eventBufferRef = useRef<EventBuffer | undefined>();
+  const [orgSwitchingState, setOrgSwitchingState] = useState<OrgSwitchingState>(
+    {
+      state: "idle",
+    }
+  );
 
+  /**
+   * Storage key for the User token (current organization).
+   */
+  const STORAGE_TOKEN_KEY = useMemo(() => NEW_STORAGE_TOKEN_KEY(oid), [oid]);
```

Comment:
This is a bit confusing as `STORAGE_TOKEN_KEY` is not a constant value, I'd suggest camelcase here